tests/src/test/scala/cats/tests/ParallelSuite.scala

```diff
@@ -303,22 +303,6 @@ class ParallelSuite extends CatsSuite with ApplicativeErrorForEitherTest {
     }
   }
 
-  test("ParMap over Stream should be consistent with zip") {
```

Comment:
I couldn't find the Stream version of this test any more?

Reply:
Oh, ouch, that's my fault. Fixed now.

Reply:
Well, not quite, one second…